docsets/ARM_A64_ISA/arm64.tgz

```diff
@@ -0,0 +1 @@
+versions/2020.12/arm64.tgz
```

Comment:
Symlinks are not supported. Please include the archive again, even if it's a duplicate.